code/_js/menus.dm

```diff
@@ -4,21 +4,21 @@ function dropdowns() {
 	var headers = new Array();
 	var links = new Array();
 	for(var i=0;i<divs.length;i++){
-		if(divs\[i\].className=='header') {
+		if(divs\[i\].className == 'header') {
 			divs\[i\].className='header closed';
 			divs\[i\].innerHTML = divs\[i\].innerHTML+' +';
 			headers.push(divs\[i\]);
 		}
-		if(divs\[i\].className=='links') {
+		if(divs\[i\].className == 'links') {
 			divs\[i\].className='links hidden';
 			links.push(divs\[i\]);
 		}
 	}
 	for(var i=0;i<headers.length;i++){
-		if(typeof(links\[i\])!== 'undefined' && links\[i\]!=null) {
+		if(typeof(links\[i\]) !== 'undefined' && links\[i\] != null) {
```

Comment:
random aside, the fact that github knows dm syntax well enough to know it does not require you escape unmatched closing brackets in strings and it highlights these as unneeded, and its done so for the last 5 to 7 years, still impresses me. even if its useless and unneeded since keeping these symmetrical is worth it.